README.md

```diff
@@ -43,120 +36,74 @@ Check the original technical report
 
 | Field Name       | Type                                        | Description                                                            |
 |------------------|---------------------------------------------|------------------------------------------------------------------------|
-| dlm:data         | [Data Object](#data-object)                 | Describes the EO data compatible with the model.                       |
-| dlm:inputs       | [Inputs Object](#inputs-object)             | Describes the transformation between the EO data and the model inputs. |
-| dlm:architecture | [Architecture Object](#architecture-object) | Describes the model architecture.                                      |
-| dlm:runtime      | [Runtime Object](#runtime-object)           | Describes the runtime environments to run the model (inference).       |
-| dlm:outputs      | [Outputs Object](#outputs-object)           | Describes each model output and how to interpret it.                   |
+| dlm:input        | [ModelInput](#model-input)               | Describes the transformation between the EO data and the model input.  |
+| dlm:architecture | [Architecture](#architecture) | Describes the model architecture.                                      |
+| dlm:runtime      | [Runtime](#runtime)           | Describes the runtime environments to run the model (inference).       |
+| dlm:output       | [ModelOutput](#model-output)             | Describes each model output and how to interpret it.                   |
```

Comment:
Totally agree on supporting multiple inputs/outputs.

I think calling it Model Input Object might make more sense though. calling it "Model Inputs" might imply only multiple model inputs are supported. when there is only one tensor it might be confusing. "model input" could refer to the single array that might contain one or more inputs. What do you think? I don't feel too strongly about this so I'll leave it to your call.

I'll add back "Object", it initially seemed redundant to me but I'll add those in if it helps remove ambiguity.

Reply:
I think an array of Model Input objects would work here . this would allow elevation to have different normalization statistics then a reflectance image which is wrapped in it's own Model Input Object.

Reply:
I like the array (`[ModelInput]`) suggestion.